.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This setting shouldn't be in repository. Each developer should take care that his IDE stuff will not appear in repository.
All what you need to do is:
In `~/.gitconfig` set:
```
[core]
        excludesFile = ~/.gitignore
```
and in `~/.gitignore` add
```
.vscode
```